### PR TITLE
[Shifts3] XML export

### PR DESCRIPTION
**Pull request by orenyk**
Implements XML export for WorkDay integration

### CODE REVIEW

**Comment by orenyk** [comment]
I think this can safely be merged, @njlxyaoxinwei take a second look through and if you're happy merge away!

File: app/controllers/payform_sets_controller.rb [hunk 1]
```diff
@@ -12,14 +12,19 @@ def show
     @payform_set = PayformSet.find(params[:id])
     payform_items = @payform_set.payforms.map(&:payform_items).flatten
     @grouped_items = payform_items.group_by{|pi| pi.category.name}
-
+    @sorted_payforms = @payform_set.payforms.delete_if{|payform| payform.hours == 0}.sort_by{|payform| payform.user.last_name}.sort_by{|payform| payform.date}
```
Comment (mnquintana) [inline app/controllers/payform_sets_controller.rb]: That is a loooooooong line

File: app/views/payform_sets/_payform.xml.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        <Cell ss:Index="18"><Data ss:Type="String">N</Data></Cell>
+        <Cell ss:Index="24"><Data ss:Type="String"><%= ptaeoO2 %></Data></Cell>
+        <Cell ss:Index="27"><Data ss:Type="Number">1</Data></Cell>
+        <Cell><Data ss:Type="String">Related_Calculation_ID</Data></Cell>
+        <Cell><Data ss:Type="String">BASERT</Data></Cell>
+        <Cell><Data ss:Type="Number"><%= payform.payrate %></Data></Cell>
+    </Row>
+ <% end %>
+
+
```
Comment (mnquintana) [inline app/views/payform_sets/_payform.xml.erb]: One too many newlines?

File: app/controllers/payform_sets_controller.rb (hunk 1 above)
Comment (orenyk) [inline app/controllers/payform_sets_controller.rb]: Yea I know, I figured we'd do a `rubocop` purge eventually so didn't want to be picky.

File: app/views/payform_sets/_payform.xml.erb (hunk 2 above)
Comment (orenyk) [inline app/views/payform_sets/_payform.xml.erb]: See above :-)

File: app/controllers/payform_sets_controller.rb (hunk 1 above)
Comment (mnquintana) [inline app/controllers/payform_sets_controller.rb]: Fair enough :smile:

**Pull request merged**